README.md

```diff
@@ -1,32 +1,43 @@
-# @inrupt/solid-client
+# Solid JavaScript Client - solid-client-js
+@inrupt/solid-client is a JavaScript library for accessing data and managing permissions on data stored in Solid Pods. It provides an abstraction layer on top of both Solid and Resource Description Framework (RDF) principles and is compatible with the RDF/JS specification. You can use solid-client in Node.js using CommonJS modules and in the browser with a bundler like Webpack, Rollup, or Parcel.
 
-# Installation
-
-```bash
-npm install @inrupt/solid-client solid-auth-client
-```
+# Inrupt Solid JavaScript Client Libraries
+Inrupt Solid JavaScript Client Libraries are a group of libraries designed to support developers building Solid applications. solid-client provides an abstraction layer on top of both Solid and Resource Description Framework (RDF) principles and is compatible with the RDF/JS specification.
 
-# Usage
+## Authentication - solid-client-authn
+[@inrupt/solid-client-authn](https://github.com/inrupt/solid-client-authn) allows developers to authenticate against a Solid server. This is necessary when the resources on your Pod are not public..
 
-See https://inrupt.github.io/solid-client-js/docs/
+## Vocabularies and interoperability - solid-common-vocab-rdf
+[@inrupt/solid-common-vocab-rdf](https://www.npmjs.com/package/@inrupt/solid-common-vocab). allows developers to build interoperable apps by reusing well-known vocabularies. This library provides vocabularies available as constants that you just have to import.
```

Comment:
@NSeydoux correct me if I'm wrong, but IIRC this is not a single library. Would this be more accurate?

```suggestion
[@inrupt/solid-common-vocab-rdf](https://github.com/inrupt/solid-common-vocab-rdf) allows developers to build interoperable apps by reusing well-known vocabularies. These libraries provide vocabularies available as constants that you just have to import.
```

Reply:
Correct, leaving the GH repo https://github.com/inrupt/solid-common-vocab-rdf (as I think is the latest change) would be simpler, this repo will then point to the artifacts on the different repos (NPM, Maven, etc.).